OPHD/States/CrimeExecution.cpp

```diff
@@ -44,3 +47,29 @@ void CrimeExecution::stealFood(FoodProduction& structure)
 			NotificationArea::NotificationType::Warning);
 	}
 }
+
+
+void CrimeExecution::stealRefinedResources(Structure& structure)
+{
+	if (structure.storage().isEmpty())
+	{
+		return;
+	}
+
+	auto resourceIndicesWithStock = structure.storage().getIndicesWithStock();
+
+	// TODO: Rob randomly from stock, for now it just robs from first available indice with resources
+	auto indexToStealFrom = resourceIndicesWithStock[0];
```

Comment:
This is quite reasonable for initial review. I'd probably want to fix it before merging though.

Reply:
Sounds good. Thanks for unblocking the RandomNumberGenerator. I work on getting it finished tomorrow to unblock this PR.